dspace/config/emails/validation_orcid

```diff
@@ -0,0 +1,22 @@
+## E-mail sent to DSpace users when they confirm the orcid email address for the account
+##
+## Parameters: {0} is expanded to a special registration URL
+##
+## See org.dspace.core.Email for information on the format of this file.
+##
+#set($subject = "${config.get('dspace.name')} Account Registration")
+#set($phone = ${config.get('mail.message.helpdesk.telephone')})
+To confirm your email and create the needed account, please click the link
+below:
+
+  ${params[0]}
+
+If you need assistance with your account, please email
+
+  ${config.get("mail.helpdesk")}
+#if( $phone )
+
+or call us at ${phone}.
+#end
+
+The DSpace-CRIS Team
```

Comment:
Same here, this should say `The ${config.get('dspace.name')} Team`, like all other email templates